theano/sandbox/cuda/opt.py

```diff
@@ -1902,7 +1895,7 @@ def local_gpu_downsample_factor_max(node):
             return
         x, = node.inputs
         if (x.owner and isinstance(x.owner.op, HostFromGpu)):
-            gpu_ds = GpuDownsampleFactorMax(node.op.ds, node.op.ignore_border)
+            gpu_ds = GpuDownsampleFactorMax(**node.op._props_dict())
```

Comment:
@nouiz I guess there's been a mistake here. I'm supposed to change only the GPUOps but this isn't an OP, right ?